.github/workflows/generate-sdk.yml

```diff
@@ -89,6 +89,7 @@ jobs:
         uses: actions/download-artifact@v2
       - name: Add and Commit files
         run: |
+          echo ${{ github.head_ref }}
```

Comment:
this needs to be removed right? as you commented on my auditapi mr

Reply:
removed